library/std/src/sys/unix/process/process_common.rs

```diff
@@ -75,11 +76,14 @@ pub struct Command {
     args: Vec<CString>,
     argv: Argv,
     env: CommandEnv,
+    env_size: Option<usize>,
+    arg_max: Option<isize>,
```

Comment:
Can `arg_max` possibly be `0`?

Reply:
It wouldn't make sense to be that. Are you suggesting a naked isize?

Reply:
Either that or `NonZero` types.

Reply:
That looks cool. Let me get it changed.